Sources/Tabs/HomeCoordinator.swift

```diff
@@ -8,13 +8,24 @@ final class HomeCoordinator: NSObject,
     var parentCoordinator: ParentCoordinator?
     var root = UINavigationController()
     private var accessToken: String?
-    private let baseVc = TokensViewController()
+    private var baseVc: TokensViewController?
 
     func start() {
-        root.setViewControllers([baseVc], animated: true)
+        let tokensViewController = TokensViewController(TokensViewModel {
+            self.showDeveloperMenu()
+        })
```

Comment:
Dya mind if we initialise the view model on the line above the view controller and pass in? Syntax here is a bit funky to read